### PR TITLE
Fix quick check failure

### PR DESCRIPTION
**Pull request by liya2017**
Fix issues as below:

1. Remove the Latest_Linters steps from ci-success check since it allow failure
2. Fixed make check-dirty-rpc-doc failure

### CODE REVIEW

**Comment by nervos-bot-user** [comment]
Can one of the admins verify this patch?

**Comment by nervos-bot-user** [comment]
Can one of the admins verify this patch?

**Review by driftluo — COMMENTED** [review]
I suggest quick_check/linter/wasm/license to run first, then run unit test/integration test, and finally, run the latest linter

File: Makefile [hunk 1]
```diff
@@ -54,7 +54,8 @@ doc-deps: ## Build the documentation for the local package and all dependencies.
 
 .PHONY: gen-rpc-doc
 gen-rpc-doc:  ## Generate rpc documentation
-	rm -f target/doc/ckb_rpc/module/trait.*.html
+
+	rm -f ${CARGO_TARGET_DIR}/doc/ckb_rpc/module/trait.*.html
```
Comment (yangby-cryptape) [inline Makefile:58]: Could not work as expected, because:
https://github.com/nervosnetwork/ckb/blob/28272b0197151c20a27b0797f0e2510a7b685c4f/Makefile#L10

**Review by yangby-cryptape — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -7,7 +7,7 @@ CLIPPY_OPTS := -D warnings -D clippy::clone_on_ref_ptr -D clippy::enum_glob_use
 	-A clippy::mutable_key_type -A clippy::upper_case_acronyms
 CKB_TEST_ARGS := ${CKB_TEST_ARGS} -c 4
 INTEGRATION_RUST_LOG := info,ckb_test=debug,ckb_sync=debug,ckb_relay=debug,ckb_network=debug
-CARGO_TARGET_DIR ?= "../target"
+CARGO_TARGET_DIR ?= "target"
```
Comment (yangby-cryptape) [inline Makefile]: The follow line could not work as expected.

https://github.com/nervosnetwork/ckb/blob/28272b0197151c20a27b0797f0e2510a7b685c4f/Makefile#L31

**Review by yangby-cryptape — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (liya2017) [reply Makefile:10]: please review again

File: Makefile [hunk 3]
```diff
@@ -54,7 +54,8 @@ doc-deps: ## Build the documentation for the local package and all dependencies.
 
 .PHONY: gen-rpc-doc
 gen-rpc-doc:  ## Generate rpc documentation
-	rm -f target/doc/ckb_rpc/module/trait.*.html
+	rm -f ${CARGO_TARGET_DIR}/doc/ckb_rpc/module/trait.*.html
+	[ ! -h "target" ] && ln -nsf "${CARGO_TARGET_DIR}" "target"
```
Comment (doitian) [inline Makefile:58]: Why this step is required?

File: Makefile (hunk 1 above)
Comment (doitian) [reply Makefile:57]: It's better to use absolute path for `CARGO_TARGET_DIR`.

```
CARGO_TARGET_DIR ?= $(shell pwd)/target
```

File: Makefile (hunk 3 above)
Comment (yangby-cryptape) [reply Makefile:58]: Too avoid to update `test.sh` and `rpc.py`, :dog:

File: Makefile (hunk 1 above)
Comment (liya2017) [reply Makefile:57]: If change to `CARGO_TARGET_DIR ?= $(shell pwd)/target`, should be refine run.sh at this time. Let me have a look on it.

**Review by quake — DISMISSED** [review]

**Review by driftluo — APPROVED** [review]

**Review by quake — APPROVED** [review]

**Pull request merged**